rp2040-hal/src/pwm/mod.rs

```diff
@@ -371,6 +371,38 @@ where
     pub fn set_top(&mut self, value: u16) {
         self.regs.write_top(value)
     }
+
+    /// Enable the PWM_IRQ_WRAP interrupt when this slice overflows.
+    #[inline]
+    pub fn enable_interrupt(&mut self) {
+        unsafe {
+            (*pac::PWM::ptr())
+                .inte
+                .modify(|_, w| w.bits(1 << I::DYN.num))
+        };
+    }
+
+    /// Disable the PWM_IRQ_WRAP interrupt for this slice.
+    #[inline]
+    pub fn disable_interrupt(&mut self) {
+        unsafe {
+            (*pac::PWM::ptr())
+                .inte
+                .modify(|r, w| w.bits(r.bits() & !(1 << I::DYN.num)))
+        };
+    }
+
+    /// Did this slice trigger an overflow interrupt?
+    #[inline]
+    pub fn is_overflow(&mut self) -> bool {
```

Comment:
No need for unique access:
```suggestion
    pub fn is_overflow(&self) -> bool {
```

Reply:
Good call, fixed.